src/config.js

```diff
@@ -11,5 +11,11 @@ export default {
   // How much of the buffer must be filled before we consider upswitching
   BUFFER_LOW_WATER_LINE: 0,
   MAX_BUFFER_LOW_WATER_LINE: 30,
-  BUFFER_LOW_WATER_LINE_RATE: 1
+
+  // TODO: Remove this when useBufferWaterLines is removed
+  MAX_BUFFER_LOW_WATER_LINE_NEW: 16,
```

Comment:
Not sure what we should call this, but in order to remain backwards compatible we needed a new config value or a hard coded value in `shouldSwitchToMedia_`. Perhaps a better name? I didn't really want to make it that much longer though as the name is already unwieldy.

Reply:
I think  it's probably good enough.